httpservice/Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM python:3.11-slim as tippecanoe-build-image
+FROM python:3.11-slim as map-utilities-build-image
 
 RUN apt-get update && apt-get install -y --no-install-recommends build-essential libsqlite3-dev zlib1g-dev git
 RUN git clone https://github.com/felt/tippecanoe.git && cd tippecanoe && make -j && make install && cd .. && rm -r tippecanoe
 RUN apt-get install -y wget && \
-    wget https://github.com/protomaps/go-pmtiles/releases/download/v1.11.0/go-pmtiles_1.11.0_Linux_x86_64.tar.gz && \
-    tar -xzf go-pmtiles_1.11.0_Linux_x86_64.tar.gz && \
-    rm go-pmtiles_1.11.0_Linux_x86_64.tar.gz && \
+    wget https://github.com/protomaps/go-pmtiles/releases/download/v1.11.1/go-pmtiles_1.11.1_Linux_x86_64.tar.gz && \
+    tar -xzf go-pmtiles_1.11.1_Linux_x86_64.tar.gz && \
+    rm go-pmtiles_1.11.1_Linux_x86_64.tar.gz && \
```

Comment:
Maybe in some future change we could take an approach like https://unix.stackexchange.com/questions/85194/how-to-download-an-archive-and-extract-it-without-saving-the-archive-to-disk and avoid having to change the version in 3 places.